### PR TITLE
Add support for Icelake-LP

### PR DESCRIPTION
**Pull request by idalin**

### CODE REVIEW

File: VoodooGPIO/IceLake-LP/VoodooGPIOIceLakeLP.hpp [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+    FUNCTION((char *)"i2c1", icllp_i2c1_groups),
+    FUNCTION((char *)"i2c2", icllp_i2c2_groups),
+    FUNCTION((char *)"i2c3", icllp_i2c3_groups),
+    FUNCTION((char *)"i2c4", icllp_i2c4_groups),
+    FUNCTION((char *)"uart0", icllp_uart0_groups),
+    FUNCTION((char *)"uart1", icllp_uart1_groups),
+    FUNCTION((char *)"uart2", icllp_uart2_groups),
+};
+
+//static const struct intel_pinctrl_soc_data icllp_soc_data = {
```
Comment (ben9923) [inline VoodooGPIO/IceLake-LP/VoodooGPIOIceLakeLP.hpp]: Doesn't seem necessary to keep Linux-specific code commented out :)

File: VoodooGPIO/IceLake-LP/VoodooGPIOIceLakeLP.hpp [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+//  Created by linyuzheng on 2020/8/11.
+//  Copyright © 2018 Alexandre Daoud. All rights reserved.
+//
+
+#include "../VoodooGPIO.hpp"
+
+#ifndef VoodooGPIOIceLakeLP_h
+#define VoodooGPIOIceLakeLP_h
+
+#define ICL_PAD_OWN    0x020
```
Comment (ben9923) [inline VoodooGPIO/IceLake-LP/VoodooGPIOIceLakeLP.hpp]: Please align all those definitions/macros, like on other platforms' code.

Comment (idalin) [reply VoodooGPIO/IceLake-LP/VoodooGPIOIceLakeLP.hpp]: Done

**Comment by kprinssu** [comment]
@ben9923 Approves and it looks good to me.

Merging it in, thanks @idalin.

**Pull request merged**